.travis.yml

```diff
@@ -3,15 +3,23 @@ language: node_js
 node_js:
   - "5.5.0"
 
+cache:
+  directories:
+    - node_modules
+
+addons:
+  firefox: latest-beta
+
 before_install:
   - "export DISPLAY=:99.0"
   - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
 
+install:
+  - npm config set spin false
```

Comment:
What does this do here? I thought it was an npm@2 thing only?
I think for npm@3, we disable the progress bar with [https://docs.npmjs.com/misc/config#progress](https://docs.npmjs.com/misc/config#progress) (via the [`--no-progress`](https://github.com/npm/npm/issues/8704#issuecomment-158222250) flag).